dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/DefaultArguments.java

```diff
@@ -76,12 +62,7 @@ public Class<? extends Boolean> valueType() {
 
     @Override
     public String valuePattern() {
-      StringBuilder buf = new StringBuilder();
-      Joiner joiner = Joiner.on('/');
-      joiner.appendTo(buf, V_TRUE);
-      buf.append('/');
-      joiner.appendTo(buf, V_FALSE);
-      return buf.toString();
+      return "true/t/yes/y/1/false/f/no/n/0";
```

Comment:
If possible get true and false values from the utility class itself, if that's not provided by the utility consider keeping the `V_TRUE` and `V_FALSE` arrays and having a test to confirm every value in `V_TRUE` parses as `true` (same for `V_FALSE`).

Reply:
You're right. It turns out that changing the parsing to the library call causes more trouble than the benefit of shortening the code, which was my original intent. It made some sense to keep the sample supported values in the `toString` method, but introducing the same values as the library supports and keeping them in sync promises to be a much bigger task. The existing code will be easier to maintain than my proposed change.

I'm closing the PR as it now makes little sense to me to continue working on this.